docs/developerguide.rst

```diff
@@ -504,8 +504,8 @@ Now we create a new cluster to host a new ``reana`` version (0.1.0):
 
 .. code:: console
 
-    $ minikube start --profile reana-0.1.0 --kubernetes-version="v1.11.2"
-    Starting local Kubernetes v1.11.2 cluster...
+    $ minikube start --profile reana-0.1.0 --kubernetes-version="v1.12.1" --feature-gates="TTLAfterFinished=true"
```

Comment:
If it is `1.13.1` then we need to change. Locally my working version is `1.12.4`.